.travis.yml

```diff
@@ -10,7 +10,7 @@ node_js:
 
 install:
   - npm ci
-  - npm -g install webpack-cli ts-node
+  - npm -g install webpack-cli ts-node typescript webpack
```

Comment:
Travis tests were failing for me without theses installs (`npm -g install webpack-cli ts-node typescript webpack`): 
[![Screenshot_20201104_210211](https://user-images.githubusercontent.com/42672814/98188466-0a0c8680-1ee1-11eb-8a59-e162cb75df19.png)](https://travis-ci.com/github/Luxcium/vscode-material-icon-theme/jobs/423710040#L151)

[![Screenshot_20201104_210348](https://user-images.githubusercontent.com/42672814/98188668-74bdc200-1ee1-11eb-98b6-354f8328235e.png)](https://travis-ci.com/github/Luxcium/vscode-material-icon-theme/builds/197067978#L111)

Reply:
In the last time I was more focused on GitHub Actions why Travis was slightly overlooked. But I always wanted to have a look in there where the problem is. Thanks for fixing it :)